hype-submitter_2.11/src/main/scala/com/spotify/hype/HFn.scala

```diff
@@ -10,16 +9,18 @@ trait HFn[T] extends Serializable {
   def run: T
 
   /**
-    * Override this to allow the function to modify the environment it is about to be submitted to.
-    *
-    * @param baseEnv The original environment used for the submission
-    * @return an optionally modified environment that will ultimately be used for the submission
+    * Override this to allow the function to modify the container it is about to be submitted to.
     */
-  def env(baseEnv: RunEnvironment): RunEnvironment = baseEnv
+  def image: String = HFn.defaultImage
 }
 
 object HFn {
-  def apply[T](f: => T) = new HFn[T] {
```

Comment:
Keep this constructor, so that the default image is used when just using `HFn { /* ... */ }`